src/win32/screengrab.c

```diff
@@ -4,19 +4,42 @@
 
 MMRect getScaledRect(MMRect input)
 {
-	// Configure DPI awareness to fetch unscaled display size
-	DPI_AWARENESS_CONTEXT initialDpiAwareness = SetThreadDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
-	size_t scaledDesktopWidth = (size_t)GetSystemMetrics(SM_CXSCREEN);
-	size_t scaledDesktopHeight = (size_t)GetSystemMetrics(SM_CYSCREEN);
+	// Works only from Windows 10 forward
+
+	// Configure DPI awareness to fetch unscaled display size - 
+	// DPI_AWARENESS_CONTEXT initialDpiAwareness = SetThreadDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+	// size_t scaledDesktopWidth = (size_t)GetSystemMetrics(SM_CXSCREEN);
+	// size_t scaledDesktopHeight = (size_t)GetSystemMetrics(SM_CYSCREEN);
 	// Reset DPI awareness to avoid inconsistencies on future calls to copyMMBitmapFromDisplayInRect
-	SetThreadDpiAwarenessContext(initialDpiAwareness);
-	size_t desktopWidth = (size_t)GetSystemMetrics(SM_CXSCREEN);
-	size_t desktopHeight = (size_t)GetSystemMetrics(SM_CYSCREEN);
+	// SetThreadDpiAwarenessContext(initialDpiAwareness);
+	// size_t desktopWidth = (size_t)GetSystemMetrics(SM_CXSCREEN);
+	// size_t desktopHeight = (size_t)GetSystemMetrics(SM_CYSCREEN);
+
+	// double scaleX = (double)(desktopWidth / (double)scaledDesktopWidth);
+	// double scaleY = (double)(desktopHeight / (double)scaledDesktopHeight);
+
+	// return MMRectMake(input.origin.x, input.origin.y, input.size.width / scaleX, input.size.height / scaleY);
+
+	auto activeWindow = GetForegroundWindow();
+	HMONITOR monitor = MonitorFromWindow(activeWindow, MONITOR_DEFAULTTONEAREST);
```

Comment:
**Question:** How would this work in a multi-monitor setup?

Let's say the current foreground window is on a secondary monitor, would this mean it calculates the scale factor based on this monitor?
I'm asking because libnut and nut.js do not handle multiple monitors at the moment.

Reply:
Yeah, I have tested using an app changing between my two monitors and also changing the scale factor of both together and individually and works for me